Dataset_Metrics/dataset_metrics_runner.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import papermill as pm
+import os
+
+class DatasetMetricsRunner(object):
```

Comment:
In the future I don't know if we want to orchestrate this type of analysis via a notebook vs. having a dedicated job. The latter is the approach followed by AE, that I'd like to contribute to.

This said, I like your approach of making the notebook executable!

Rule of three: we have AlgoRunner and DatasetMetricsRunner classes. If we add another Runner, we should extract an interface and generalise.

Reply:
I agree!